manage.py

```diff
@@ -37,7 +37,22 @@ def test(coverage, test_names):
         import subprocess
         os.environ['FLASK_COVERAGE'] = '1'
         sys.exit(subprocess.call(sys.argv))
+    if os.environ.get('FLASK_ENV') != 'test':
+        import subprocess
+        # ../data/test.db
+        APP_DIR = os.path.dirname(__file__)
+        TEST_PATH = os.path.abspath(os.path.join(APP_DIR, 'data', 'test.db'))
+        os.environ['FLASK_ENV'] = 'test'
+        os.environ['DATABASE_URL'] = 'sqlite:///' + TEST_PATH
+        # clear before each run
+        try:
+            os.remove(TEST_PATH)
+        except FileNotFoundError:
+            pass
+        sys.exit(subprocess.call(sys.argv))
```

Comment:
[Opinion]: The testing and production environment should be similar. We are running our tests on sqlite, but running postgres on prod.

Reply:
Each is entitled to his/her own ;-) The fixed environment is Python and Sqlalchemy. The database vendor is variable.